### PR TITLE
Make instructions for mysql backup a bit more foolproof

### PR DESCRIPTION
**Pull request by timcharper**
Just small suggestions to help others avoid making similar mistakes that I made

### CODE REVIEW

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 1]
```diff
@@ -91,18 +91,46 @@ Kaptain uses Percona Operator to back up and restore the state of MySQL database
...
+
+Type:  Opaque
+
+Data
+====
+AWS_SECRET_ACCESS_KEY:  40 bytes
+AWS_ACCESS_KEY_ID:      20 bytes
+```
+
+
```
Comment (timcharper) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:133]:
```suggestion
```

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 2]
```diff
@@ -179,13 +206,13 @@ The operation will use the latest backup data created by the backup referenced i
 To restore from a specific backup location, set the `BACKUP_RESTORE_SOURCE` variable to point to the location of the backup you want to use to restore your cluster.
 ```bash
 # Full path to the backup folder, for example, s3://mysql-store-backup/kaptain-mysql-store-2021-04-05-21:47:24-full
-export BACKUP_RESTORE_SOURCE=""
+BACKUP_RESTORE_SOURCE=""
```
Comment (timcharper) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:209]: I'm not sure why we had these exports here; they are not needed.

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 3]
```diff
@@ -91,16 +91,43 @@ Kaptain uses Percona Operator to back up and restore the state of MySQL database
 You must create a Kubernetes `Secret` with AWS access credentials for the backup and then update the Kaptain configuration to enable backup and configure the storage location.
 
 Create a Kubernetes `Secret` in the same namespace where Kaptain is installed using the AWS credentials as follows:
-```yaml
+```bash
+AWS_ACCESS_KEY_ID=<aws_access_key_id>
```
Comment (gflorinskaya) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:95]:
```suggestion
AWS_ACCESS_KEY_ID="<aws_access_key_id>"
AWS_SECRET_ACCESS_KEY="<aws_secret_access_key>"
WORKSPACE_NAMESPACE="<workspace_namespace>"
```

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 4]
```diff
@@ -91,16 +91,43 @@ Kaptain uses Percona Operator to back up and restore the state of MySQL database
...
+AWS_SECRET_ACCESS_KEY=<aws_secret_access_key>
```
Comment (gflorinskaya) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:96]:
```suggestion
AWS_SECRET_ACCESS_KEY="<aws_secret_access_key>"
```

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 5]
```diff
@@ -91,16 +91,43 @@ Kaptain uses Percona Operator to back up and restore the state of MySQL database
...
+WORKSPACE_NAMESPACE=<workspace_namespace>
```
Comment (gflorinskaya) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:97]:
```suggestion
WORKSPACE_NAMESPACE="<workspace_namespace>"
```

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 6]
```diff
@@ -117,7 +144,7 @@ core:
 
 To perform a backup, apply the following manifest to trigger the backup plan for the MySQL cluster component:
 ```bash
-export WORKSPACE_NAMESPACE=<workspace_namespace>
+WORKSPACE_NAMESPACE=<workspace_namespace>
```
Comment (gflorinskaya) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:147]:
```suggestion
WORKSPACE_NAMESPACE="<workspace_namespace>"
```

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md (hunk 2 above)
Comment (gflorinskaya) [reply pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:209]: The difference between "export" vs "no export" is explained here:
https://www.baeldung.com/linux/bash-variables-export
We do not know exact use cases and scenarios and I think keeping export is safer. It is also what we do in other parts of our documentation so we should keep it uniform

**Review by gvasquezvargas — APPROVED** [review]

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md (hunk 2 above)
Comment (timcharper) [reply pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:209]: Having done a fair amount of shell scripting in my days, I can safely say that in these instructions, there is absolutely no reason to export them. Personally, it feels messy to export variables unnecessarily. To me, it's a bit like adding an extra step in documentation that does absolutely nothing to achieve your end goal.

However, it is relatively harmless here so I can understand a desire to keep a consistent style. This isn't a hill I will choose to die on.

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 7]
```diff
@@ -179,13 +206,13 @@ The operation will use the latest backup data created by the backup referenced i
...
 ```bash
 # Full path to the backup folder, for example, s3://mysql-store-backup/kaptain-mysql-store-2021-04-05-21:47:24-full
-export BACKUP_RESTORE_SOURCE=""
+BACKUP_RESTORE_SOURCE=""
 # Secret name with AWS credentials, for example, "mysql-backup-secret"
-export BACKUP_CREDENTIALS_SECRET=""
+BACKUP_CREDENTIALS_SECRET=""
 # Bucket region, for example, "us-west-2"
-export BACKUP_BUCKET_REGION=""
+BACKUP_BUCKET_REGION=""
```
Comment (timcharper) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:213]:
```suggestion
export BACKUP_BUCKET_REGION=""
```

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md (hunk 2 above)
Comment (timcharper) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:209]:
```suggestion
export BACKUP_RESTORE_SOURCE=""
```

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md (hunk 3 above)
Comment (timcharper) [reply pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:95]: If you need quotes around your AWS access key id, you have done something wrong. These are always alphanumeric.

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 8]
```diff
@@ -117,7 +144,7 @@ core:
 
 To perform a backup, apply the following manifest to trigger the backup plan for the MySQL cluster component:
 ```bash
-export WORKSPACE_NAMESPACE=<workspace_namespace>
+WORKSPACE_NAMESPACE="<workspace_namespace>"
```
Comment (timcharper) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:147]:
```suggestion
export WORKSPACE_NAMESPACE="<workspace_namespace>"
```

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 9]
```diff
@@ -91,16 +91,43 @@ Kaptain uses Percona Operator to back up and restore the state of MySQL database
...
-  AWS_ACCESS_KEY_ID: <base64-enconded AWS Access Key ID>
-  AWS_SECRET_ACCESS_KEY: <base64-enconded AWS Secret Access Key>
+  AWS_ACCESS_KEY_ID: $(echo -n "$AWS_ACCESS_KEY_ID" | base64)
+  AWS_SECRET_ACCESS_KEY: $(echo -n "$AWS_SECRET_ACCESS_KEY" | base64)
+EOF
+```
+
+Confirm that your secret is configured correctly:
+
+```bash
```
Comment (timcharper) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:113]:
```suggestion
```sh
```

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 10]
```diff
@@ -91,16 +91,43 @@ Kaptain uses Percona Operator to back up and restore the state of MySQL database
 You must create a Kubernetes `Secret` with AWS access credentials for the backup and then update the Kaptain configuration to enable backup and configure the storage location.
 
 Create a Kubernetes `Secret` in the same namespace where Kaptain is installed using the AWS credentials as follows:
-```yaml
+```bash
```
Comment (timcharper) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:94]:
```suggestion
```sh
```

**Review by alembiewski — APPROVED** [review]

File: pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md [hunk 11]
```diff
@@ -91,16 +91,43 @@ Kaptain uses Percona Operator to back up and restore the state of MySQL database
...
+EOF
+```
+
+Confirm that your secret is configured correctly:
+
+```sh
+kubectl describe secret mysql-backup-secret -n ${WORKSPACE_NAMESPACE}
+```
+
+The output should look like this:
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md:117]:
```suggestion
The output should be similar to this:
```

**Review by lukeogg — APPROVED** [review]
LGTM

**Pull request merged**